qiskit/visualization/circuit/circuit_visualization.py

```diff
@@ -225,6 +225,11 @@ def circuit_drawer(
 
     if idle_wires is None:
         idle_wires = default_idle_wires
+    elif isinstance(idle_wires, str):
```

Comment:
```suggestion
    if isinstance(idle_wires, str):
```

still need to handle `auto` if `idle_wires` was input as `None` - line 227 would set it to `"auto"` and then it'd skip this block that resolves it

Reply:
whoops my bad!